quantum/rgb_matrix/animations/pixel_fractal_anim.h

```diff
@@ -67,7 +67,7 @@ static bool PIXEL_FRACTAL(effect_params_t* params) {
         led[h][MID_COL] = led[h][MID_COL - 1] = (random8() & 3) ? false : true;
     }
 
-    wait_timer = g_rgb_timer + interval();
+    wait_timer = timer_read32() + interval();
```

Comment:
```suggestion
    wait_timer = g_rgb_timer + interval();
```

Reply:
@zvecr  got it. I've reverted the file changes to previous commit and I'm going to create a new branch for those changes. Thanks!